src/frontend/src/lib/components/wizards/auth/AuthWizard.svelte

```diff
@@ -44,7 +46,11 @@
     try {
       onSignIn(await authFlow.continueWithExistingPasskey());
     } catch (error) {
-      onError(error);
+      if (isWebAuthnCancelError(error)) {
+        throw error; // Error is handled by child component
```

Comment:
I didn't understand this at first.

Could we say: "This function is called in the child component and this error is handled there"